prql-compiler/src/sql/un_group.rs

```diff
@@ -0,0 +1,33 @@
+use anyhow::Result;
+
+use crate::ast::ast_fold::*;
+use crate::ast::*;
+
+pub fn un_group(nodes: Vec<Node>) -> Result<Vec<Node>> {
+    UnGrouper {}.fold_nodes(nodes)
+}
+
+/// Traverses AST and replaces transforms with nested pipelines with the pipeline
+struct UnGrouper {}
```

Comment:
I'm trying to think of a better name, so far unsuccessfully...

Reply:
UnNester? DeNester? Flatten?